### PR TITLE
[DO NOT MERGE] Testing new settings for test automation

### PR DESCRIPTION
**Pull request by johnpmitsch**
Just a test of new ansible settings in test automation seeing if we get consistent results

### CODE REVIEW

**Comment by johnpmitsch** [comment]
yee-haw

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Satellite-clone
 
-Easily set up a Red Hat Satellite server with restored backup data.
+VERY Easily set up a Red Hat Satellite server with restored backup data.
```
Comment (jturel) [inline README.md]: Lol

**Comment by johnpmitsch** [comment]
Got a green run! going for 4 in a row

**Comment by johnpmitsch** [comment]
yee-haw

**Comment by jturel** [comment]
:game_die:

**Comment by johnpmitsch** [comment]
Another green! For the sake of thoroughness I'm kicking one more off, but looks like we have a more reliable testing solution. :turtle:

**Comment by johnpmitsch** [comment]
yee-haw

**Comment by johnpmitsch** [comment]
Still green!
h

**Pull request closed without merging**